README.md

```diff
@@ -1,24 +1,241 @@
+-->
 
-* Database creation
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Database initialization
+  <h3><b>Microverse README Template</b></h3>
 
-* How to run the test suite
+</div>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- TABLE OF CONTENTS -->
 
-* Deployment instructions
+# 📗 Table of Contents
 
-* ...
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog of Things ](#-catalog-of-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails Social Media <a name="about-project"></a>
+
+> in this rails project I will be making a small social media in which people can like, comment and share posts
+
+**Rails Social Media** is an app in which you can share posts and other users including yoursel can like and comment, each user has their own profile in which shows some details about the user including the posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech stack and relevant sections that apply to the project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> 1-3 key features of the application.
+
+- **Record Posts**
+- **Record Users**
+- **Record Comments**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to the deployed project.
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd my-folder
+  git clone https://github.com/didierganthier/ruby-group-capstone
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly make sure the readme file is describing the exact project you are working on. This will make your readme file more professional. I think you are supposed to clone `rails-social-media` instead of `ruby-group-capstone`.